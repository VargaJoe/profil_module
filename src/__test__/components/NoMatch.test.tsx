import * as React 						  	from 'react';
import NoMatch 							    from '../../components/NoMatch';
import { 
	configure, 
		shallow,
		mount } 							from 'enzyme';
import * as Adapter 					  	from 'enzyme-adapter-react-16';
import toJson 				      			from 'enzyme-to-json';

configure( {adapter: new Adapter()} );

describe('<NoMatch /> shallow rendering', () => {
		
	const nomatch = shallow(<NoMatch />); 
		
	// test Snapshot 
	it('Match to snapshot', () => {
		expect(toJson(nomatch)).toMatchSnapshot();
	});
		
});
