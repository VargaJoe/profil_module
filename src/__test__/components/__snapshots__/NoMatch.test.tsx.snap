// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> shallow rendering Match to snapshot 1`] = `
<div
  className="no_mathc"
>
  <h1>
    404
  </h1>
  <p>
    Not found
  </p>
  <Link
    replace={false}
    to="/"
  >
    Back to home
  </Link>
</div>
`;
